src/_community/rules.md

```diff
@@ -46,14 +46,26 @@ Essentially, please take requests from community team members seriously and list
 something. If you have a problem with what a team member is asking you to do, we advise that you step away from the 
 situation and send a private message to another team member.
 
-#### 5) Keep all projects legal & appropriate
-
-If you need help with a project, you will not be provided with it if your project may break laws, breach a Terms of
-Service or EULA (belonging to any organisation, not just the platforms we're using), or be considered malicious or 
-inappropriate.
-
-Additionally, keep your code appropriate. Do not sexualize your code or use any slurs or other forms of derogatory
-language, either in the code itself or in the assets or filenames that make up your project.
+#### 5) Keep all projects legal, legitimate & appropriate
+
+While anyone may make use of the Quilt toolchain in accordance with the licenses applied to each project, all projects 
+developed or promoted **in all Quilt community spaces** must meet the following criteria:
+
+* Projects must not be malicious - which includes (but is not limited to) malware, mods that include backdoors, mods
+  that modify Quilt in an attempt to hide themselves from servers they connect to, or mods that harass specific people 
+  or groups of people (for example, by automatically kicking them from servers with the mod installed)
+* Projects must not be sexualised, hateful or otherwise inappropriate in nature, and this also applies to all code 
+  and project assets posted for any purpose
+* Projects must not break laws, including copyright laws - this includes piracy tools (including mods that use 
+  projects like `youtube-dl` for in-game music streaming) and hacked clients, as well as projects that fail to adhere 
```

Comment:
> * Projects must not break laws, including copyright laws - this includes piracy tools (including mods that use 
  projects like `youtube-dl` for in-game music streaming) and hacked clients, as well as projects that fail to adhere 
  to the licenses of other projects to the best of their ability

Which laws do hacked clients break?
I'm sure some hacked clients break the law (perhaps due to licensing or similar), however there is plenty of scope for hacked clients that do not break the law...

Reply:
Also perhaps the exact jurisdictions who's laws cannot be broken should be clarified? I am sure people from many different jurisdictions will be the intended users of quilt...

Reply:
> Which laws do hacked clients break?
> I'm sure some hacked clients break the law (perhaps due to licensing or similar), however there is plenty of scope for hacked clients that do not break the law...

The colloquial definition of "hacked client" (which is probably not what the client devs themselves use) is, to the best of my memory, clients which combine a pirated copy of Minecraft with a bunch of cheat/utility mods.

> Also perhaps the exact jurisdictions who's laws cannot be broken should be clarified? I am sure people from many different jurisdictions will be the intended users of quilt.

We're not lawyers, and as you quite rightly noted, Quilt will likely have users and developers from all over the world - this part of the rule is more about intent, and what most people understand to be illegal - copyright violations and malware, for example.

Trying to specify specific jurisdictions is futile, especially considering the community team (and the rest of the Quilt teams) is not based in any specific location.

Reply:
> The colloquial definition of "hacked client" (which is probably not what the client devs themselves use) is, to the best of my memory, clients which combine a pirated copy of Minecraft with a bunch of cheat/utility mods.

Ok, that makes sense, perhaps this should be widened to simply the distribution of Minecraft's code? As you said, hacked client is often used a bit ambiguously.

> We're not lawyers, and as you quite rightly noted, Quilt will likely have users and developers from all over the world - this part of the rule is more about intent, and what most people understand to be illegal - copyright violations and malware, for example.

Not every country have the same copyright laws. If good intent is the intent of this rule, perhaps the rule should just be about good intent - requiring mods to respect the rights and property of others for example. Bringing the law into it is confusing.

Reply:
Unfortunately, I don't think boiling it down to intent entirely is workable - you'll find most services define something like this the same way, and I think that's largely because it's something you may need to think about later on. From Discord's ToS:

> As an example, you agree not to use the Service in order to:
> 
> [...]
> 
> engage in conduct that is fraudulent or illegal or otherwise harmful to Discord or any other user;

Additionally,

> You specifically acknowledge that the Company shall not be liable for user content, including without limitation Your Content, or the defamatory, offensive, or illegal conduct of any third party and that the risk of harm or damage from the foregoing rests entirely with you.

Life's a bitch, ain't it?

Reply:
I don't think companies such as discord should be used as role models. Their terms are written strictly to benefit their own interests rather than those of their users. 
Anyway, if you want to stay with enforcing legal usage I at least think you should specify which jurisdiction.
Simply saying that American federal law should be followed would be alright (although still quite unfair to international users).

Reply:
As a European, I doubt I could easily speak on US federal law. On the other hand, I doubt a US-centric community team member could easily speak on GDPR or other European laws. It ain't that simple - although yes, we all speak English, so it's at least a somewhat Europe-and-US-centered approach.

Specifying a jurisdiction would, in my opinion, require people who are experts in that jurisdiction. Not doing so means we can provide a best-effort attempt, based on the individual knowledge levels of each staff member.

It's a tricky one to think around - but on the other hand, I don't think I've actually seen communities like this specify jurisdictions. Considering Discord pushes the legal burden on to us, it probably makes the most sense to take the same approach they do when defining this kind of rule.

I'm open to suggestions that address this specific mindset, though.

---

> I don't think companies such as discord should be used as role models. Their terms are written strictly to benefit their own interests rather than those of their users.

This is 100% verifiably true - the point was more that it's restrictive enough that it's unlikely that other platforms we use will be more restrictive.

Reply:
> Specifying a jurisdiction would, in my opinion, require people who are experts in that jurisdiction. Not doing so means we can provide a best-effort attempt, based on the individual knowledge levels of each staff member.

Not specifying one means people have to make an attempt to follow undefined rules. How do I follow undefined rules? I assume I would have to make some assumptions about the communities stance on topics like intellectual property, computer usage and harassment. But obviously some of these topics are very divisive and there might not be any clear consensus in the community. It is more likely to come down to the views of whichever moderator happens to be online.

If you are not able to define what is "legal" it is unfair to make other abide by it. Instead define the actual rules you are thinking of, for example intellectual property, computer usage and harassment. Many of these are already defined in your pull request.

And yes, I understand your desire to follow discord's rules. I do not however see any need to enforce these rules on the entire community. I can understand that the discord part of the community, and the organisation (incl its public figures) should follow these rules. However not the entire community.

Reply:
I agree that it's not ideal, but I'm not sure how best to approach it. I'll workshop it internally a bit.

Reply:
Having chatted a bit, this doesn't seem to be a useful argument - Discord doesn't define a jurisdiction, so the best we can do is follow their definition and moderate to the best of our ability.

That said, this doesn't have to be iron-fisted, and largely much of our enforcement doesn't work that way - if someone didn't know something was problematic, I'd be happy to give them a second chance in most situations.

You're not wrong that this rule repeats itself a lot - I consider that useful for clarity, personally. Either way, I feel like the current wording is the best middle ground in this case.